src/jmclient/wallet_utils.py

```diff
@@ -1518,9 +1518,7 @@ def open_wallet(path, ask_for_password=True, password=None, read_only=False,
 
     if not Storage.is_storage_file(path):
         raise Exception("Failed to open wallet at '{}': not a valid joinmarket"
-                        " wallet.\n\nIf this wallet is in the old json format "
```

Comment:
maybe could also just rephrase it:
"if this wallet is in the old json format, you need an older joinmarket version to convert this wallet to the new format"

Reply:
Don't think that's necessary. New wallet format was introduced more than six years ago with v0.4.0, before even adding any native segwit support to JoinMarket. Every P2WPKH wallet is guaranteed to not use old format. With such old wallets it makes more sense to just send all the coins to fresh native segwit wallet. But I'm not sure such old wallets still even exist at all, as both legacy and wrapped segwit JoinMarket coinjoin market have been dead for a long time.